packages/react-components/react-menu/library/src/components/Menu/Menu.types.ts

```diff
@@ -18,8 +18,8 @@ export type MenuProps = ComponentProps<MenuSlots> &
     'checkedValues' | 'defaultCheckedValues' | 'hasCheckmarks' | 'hasIcons' | 'onCheckedValueChange'
   > & {
     /**
-     * Can contain two children including {@link MenuTrigger} and {@link MenuPopover}.
-     * Alternatively can only contain {@link MenuPopover} if using a custom `target`.
+     * Can contain two children including MenuTrigger and MenuPopover.
+     * Alternatively can only contain MenuPopover if using a custom `target`.
```

Comment:
_nit:_ You could in its place use backticks to make the text look different

```suggestion
     * Can contain two children including `MenuTrigger` and `MenuPopover`.
     * Alternatively can only contain `MenuPopover` if using a custom `target`.
```